contrib/test_jenkins.sh

```diff
@@ -218,9 +218,10 @@ if [ -n "$JENKINS_RUN_TESTS" ]; then
     rm -f jenkins_sidelinks.txt
     echo 1..1 > coverity.tap
     if [ $nerrors -gt 0 ]; then
+        cov-format-errors --dir $cov_build --emacs-style
         echo "not ok 1 Coverity Detected $nerrors failures # $cov_url" >> coverity.tap
     else
-        echo ok 1 Coverity found no issues >> coverity.tap
+        echo "ok 1 Coverity found no issues" >> coverity.tap
```

Comment:
not realy needed, it's just for consistency with previous line, i can remove it if it's very critical